.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Exclude this file from changes pushed to development please :)

Reply:
Cool, get on it now. Gracias!

Reply:
done